src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dialogs/HelpDialog.java

```diff
@@ -1,7 +1,136 @@
+public class HelpDialog extends Stage {
+    private Font sectionFont = new Font(20);
+    private static final double WINDOW_HEIGHT = 400;
+    private static final double WINDOW_WIDTH = 600;
+
+    /**
+     * Creates a new HelpDialog
+     */
+    public HelpDialog() {
+        setTitle(Translator.getInstance().getString("monitoring.helpdialog.title"));
+        getIcons().add(new Image("/icon.png"));
+
+        TabPane tabPane = new TabPane();
+        tabPane.getTabs().addAll(createMainTab(), createSettingsTab());
+
+        Button close = new Button(Translator.getInstance().getString("monitoring.helpdialog.closeButton"));
+        close.setDefaultButton(true);
+        close.setCancelButton(true);
+        close.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent actionEvent) {
+                Stage stage = (Stage) close.getScene().getWindow();
+                stage.close();
+            }
+        });
+
+        BorderPane pane = new BorderPane();
+        pane.setCenter(tabPane);
+        pane.setBottom(close);
+        pane.setAlignment(close, Pos.CENTER);
+        setScene(new Scene(pane));
+        setHeight(WINDOW_HEIGHT);
+        setWidth(WINDOW_WIDTH);
+    }
+
+    private Tab createMainTab() {
+        Tab tab = new Tab();
+        tab.setText(Translator.getInstance().getString("monitoring.helpdialog.mainTab.header"));
+
+        TextFlow textFlow = new TextFlow();
+
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.introduction", null));
+        textFlow.getChildren().add(newLine());
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.layoutSection", sectionFont));
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.layoutText", null));
+        textFlow.getChildren().add(newLine());
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.alarmSection", sectionFont));
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.alarmText", null));
+        textFlow.getChildren().add(newLine());
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.gaugeSection", sectionFont));
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.gaugeText", null));
+        textFlow.getChildren().add(newLine());
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.progressSection", sectionFont));
+        textFlow.getChildren().add(getText("monitoring.helpdialog.mainTab.progressText", null));
+
+        ScrollPane scrollPane = new ScrollPane();
+        scrollPane.setFitToWidth(true);
+        scrollPane.setContent(textFlow);
+        scrollPane.setPrefViewportHeight(textFlow.prefHeight(-1));
+        scrollPane.setPrefViewportWidth(textFlow.prefWidth(-1));
+        scrollPane.setPadding(new Insets(5));
+
+        tab.setContent(scrollPane);
+        tab.setClosable(false);
```

Comment:
Hierfür gibt es in der TabPane-Klasse die Methode setTabClosingPolicy(). Mit dem Aufruf tabPane.setTabClosingPolicy(TabClosingPolicy.UNAVAILABLE); sind alle Tabs automatisch nicht schließbar und erspart hier und später diesen Aufruf.